src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -212,7 +212,10 @@ private static bool IsSameLocalOrField(BoundExpression expr1, BoundExpression ex
 
         private static bool IsComCallWithRefOmitted(MethodSymbol method, ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> argumentRefKindsOpt)
         {
-            if ((object)method.ContainingType == null || !method.ContainingType.IsComImport) return false;
+            if (method.ParameterCount != arguments.Length ||
```

Comment:
I thought about that and it seems that in general we want to report all the errors of this sort, so we want to visit the node. We stop early only in cases when node is broken in a way that prevents further analysis (like this case)